ports/servoshell/minibrowser.rs

```diff
@@ -190,9 +194,28 @@ impl Minibrowser {
                 });
             *toolbar_height = Length::new(height);
 
+            let scale =
+                Scale::<_, DeviceIndependentPixel, DevicePixel>::new(ctx.pixels_per_point());
+            let Some(focused_webview_id) = webviews.focused_webview_id() else {
+                return;
+            };
+            let Some(webview) = webviews.get_mut(focused_webview_id) else {
+                return;
+            };
+            let mut embedder_events = vec![];
             CentralPanel::default()
                 .frame(Frame::none())
                 .show(ctx, |ui| {
+                    let Pos2 { x, y } = ui.cursor().min;
+                    let origin = Point2D::new(x, y);
+                    let Vec2 { x, y } = ui.available_size();
+                    let size = Size2D::new(x, y);
+                    let rect = Rect::new(origin, size) * scale;
```

Comment:
```suggestion
                    let Pos2 { x, y } = ui.cursor().min;
                    let Vec2 { x, y } = ui.available_size();
                    let rect = Rect::new(
                        Point2D::new(x, y),
                        Size2D::new(width, height)
                     ) * scale;
```

Reply:
Fixed in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)